### DISCUSSION TITLE
You dont have access to this Project; Rowy Run

### DISCUSSION THREAD

**Discussion by talkd** [AntlerVC/firetable#882] [Support / Q&A] [answered]
Hi,

Somehov I am not allowed to implement Rowy Run. When I log into my google account I am told that I dont have acces to the project - which is kind of weird since I have acces to Firebase in the other part of Rowy?

**Comment by BeeBombshell** [comment]
Hey @talDigital, I'd suggest making sure that you are the owner of the project you're trying to enable Rowy Run on since only the owner role has the permissions for this. If not so, feel free to drop a mail to our [support](support@rowy.io) with the details of the project so that we can help you further.

**Comment by shamsmosowi** [comment]
you can check your firebase roles [here](https://console.firebase.google.com/u/0/project/_/settings/iam)

**Comment by talkd** [comment, marked as answer]
Hi all,I am owner of The project. Which it is why it makes no sense. My integration with Firebase an Flutterflow works seamless. The problem occurs when I try to execute ROWY RunRegards MikkelDen 20. okt. 2022 kl. 17.11 skrev Shams ***@***.***>:﻿
you can check your firebase roles here

**Reply by talkd** [reply]
Hope somebody can help soon. My project relies on my access to Rowy 🙏

**Reply by harinij** [reply]
@talDigital this requires further investigation specifically for your account, it might be best if we can do it over a call. Could you email us at support@rowy.io so that we can book some time?
